### ISSUE TITLE
Missing other exporters configuration in the SDK configuration page

### ISSUE THREAD

**Issue by ppatierno** [open-telemetry/opentelemetry.io#1783]
I was looking at the SDK configuration documentation here https://opentelemetry.io/docs/concepts/sdk-configuration/ and noticed that other than general SDK related configs, it also lists configuration specific for the OTLP exporter.
I was wondering if it would need to list the main two other exporters as well like Jaeger and Zipkin because their configuration seems to be listed here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md but not in the documentation itself.
So someone who is using Jaeger and looking for configuration can't find content on the doc but has to go to the SDK env vars in the specification repo.

**Comment by cartermp** [comment]
In general I'd say it's fine to add these to the conceptual docs. I think at some point we're going to have to rationalize these two things:

1. The spec is comprehensive, but not intended for end users to read and/or use as a reference to learn how to do stuff
2. We'll want to keep from duplicating content if we can, and so we could pull stuff in from the spec via tooling if needed, but that also can conflict with point (1)

**Comment by ppatierno** [comment]
> The spec is comprehensive, but not intended for end users to read and/or use as a reference to learn how to do stuff

Exactly. Users should stay on the documentation website for any kind of stuff they can do with OpenTelemetry.

**Comment by siddhant-khisty** [comment]
Hey, I'd like to work on this 

If I understand it correctly, I have to move the information from the SDK environment variable from [GitHub](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md) to the [documentation page for SDK](https://opentelemetry.io/docs/concepts/sdk-configuration/).

Did I get that right? Or am I missing something?

**Comment by cartermp** [comment]
@siddhant-khisty that sounds about right, but limited only to jaeger and zipkin for now. I think we'll want to be careful about what to add here over time - there may be a better technical solution that lets us pull in text from the spec and add commentary around it, but for now I think it is fine to move some things over and change them a bit to match the existing SDK docs page.

**Comment by siddhant-khisty** [comment]
Hey, @cartermp I noticed that there is an env variable page in the documentation [here](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/). Should I add a page about it to the [SDK configuration page](https://opentelemetry.io/docs/concepts/sdk-configuration/) as well?

I made a hackmd file where I updated the current env variable files. I plan to add some more details in here. Could you please check if this file structure is alright, or does it need to be changed?
https://hackmd.io/@siddhant-khisty/rylEyfJ7i

**Comment by svrnm** [comment]
> I made a hackmd file where I updated the current env variable files. I plan to add some more details in here. Could you please check if this file structure is alright, or does it need to be changed?

@siddhant-khisty you can always raise PRs with proposed changes, even if it's not yet finished, makes it easier for us to see what has changed. In the worst case we do not merge it.

Thanks!

**Comment by cartermp** [comment]
Yes @siddhant-khisty please file a draft PR with changes if you'd like a review pass!
